### PR TITLE
deps.ffmpeg: Fix fixup step error on windows

### PR DESCRIPTION
**Pull request by Chensiyy**
### Description

Fixed fixup step file path.

### Motivation and Context

The scripts failed when I execute them.

### How Has This Been Tested?

Successfully built with WSL on my computer.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by RytoEX** [comment]
The `bin/bin` here seems like an obvious mistake and probably should be corrected (despite that we no longer cross-compile for Windows). I'm not sure about the other changes. @PatTheMav ?

**Review by PatTheMav — APPROVED** [review]
Looks good to me - we'll hopefully not use these for Windows any longer, but as long as we keep cross-compilation as an emergency fallback, it's good to have this fixed.

**Review by RytoEX — CHANGES_REQUESTED** [review]
Please add a space between each commit prefix and the rest of the commit message subject.
```
deps.ffmpeg: Fix
```

File: deps.ffmpeg/30-svt-av1.zsh [hunk 1]
```diff
@@ -111,12 +111,11 @@ fixup() {
       ;;
     windows-x*)
       if (( shared_libs )) {
-        mv (#i)"${target_config[output_dir]}"/lib/libsvtav1*.dll ${target_config[output_dir]}/bin/
```
Comment (PatTheMav) [inline deps.ffmpeg/30-svt-av1.zsh:114]: On second look this change needs to be reverted - the `rm` command further below relies on the DLL to have an all-lowercase name, I also faintly remember other issues raised from the file names being mixed case.

Comment (PatTheMav) [reply deps.ffmpeg/30-svt-av1.zsh:114]: Also the DLL still needs to be moved to the `bin` directory.

Comment (Chensiyy) [reply deps.ffmpeg/30-svt-av1.zsh]: I have reverted this commit

**Comment by Chensiyy** [comment]
> Please add a space between each commit prefix and the rest of the commit message subject.
> 
> ```
> deps.ffmpeg: Fix
> ```

I've fixed it.

**Review by RytoEX — APPROVED** [review]
Looks good to me.

**Pull request merged**